Integrations/integration-AnomaliThreatStream.yml

```diff
@@ -266,6 +266,10 @@ script:
     }
 
     var parseResult = function(command, result) {
+        if(!(result['objects'] instanceof Array)){
```

Comment:
what about `parseReputationResult`?
```
var objects = dq(result, 'objects');
objects.sort(function(x, y) {return x.created_ts < y.created_ts});
```

also - I think we should handle a case that the return value is not  an object, and not trust the api to return an object (`res.Body`)

Reply:
`parseReputationResult` is always called from `parseResult`, the bug was actually there but the source is from `parseResult`. And I didn't fully understand - did you mean the `objects` field in the result not being returned at all or the result itself not being an object?

Reply:
thanks for the `parseReputationResult` clarification

regrading the second thing - if `result` (which is the body of the returned request) is not an object - we will also get an exception here

Reply:
Added checks for `result`